### DISCUSSION TITLE
Connection Wallet

### DISCUSSION THREAD

**Discussion by rdeanda01** [Chia-Network/chia-blockchain#6938] [Support]
The connecting wallet screen does not load, What do i need to do?

**Comment by loppefaaret** [comment]
is it stuck on "Connecting to wallet..." ?
if so, you can try this:
WINDOWS WALLET DATABASE RESET

Close down the Chia app, and open up a PowerShell window, and execute the below command:
```
mv ~\.chia\mainnet\wallet\db\block* ~\Desktop\
```
Then try opening up the Chia app again
if it worked, you can remove the files we put on the Desktop!

**Comment by BrandtH22** [comment]
@rdeanda01 , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
